source/settings/core.rst

```diff
@@ -2247,6 +2251,10 @@ See :ref:`setting-login_log_format_elements`
 - Default: ``user=<%u> method=%m rip=%r lip=%l mpid=%e %c``
 - Values:  :ref:`string`
 
+.. deprecated:: v2.3.7
+
+  Use :ref:`event-auth_request_finished` event instead.
```

Comment:
I think it's overly optimistic to say that we can ever deprecate this setting. I'd just close this merge request..